package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "zendesk_app_framework_sdk",
-  "version": "2.0.37",
+  "version": "2.0.36",
```

Comment:
It was 37 should be bumped top 38?

Reply:
The latest is v2.0.36 which already has https://github.com/zendesk/zendesk_app_framework_sdk/pull/172 and https://github.com/zendesk/zendesk_app_framework_sdk/pull/174. It was `v2.0.33` and changed to `v2.0.37` by accident.

Reply:
We should probably have a process to properly bump the version when we tag zaf_sdk as part of the deployment process.

Maybe we might be able to emulate the way deployments are currently done for gems? This might be worth asking the appropriate internal team about to see if they have plans for a github action along these lines.